nvidia/cheminformatics/wf/cluster/cpukmeansumap.py

```diff
@@ -104,9 +105,9 @@ def cluster(self,
                 ml.metric_func_args = (embedding_sample, kmeans_labels_sample)
 
         with MetricsLogger('umap', self.n_molecules) as ml:
+            df_molecular_embedding = df_molecular_embedding.compute()
```

Comment:
Thanks for fixing this. Looks like it might have been instigated by some of my changes, unfortunately. :/